File: Solutions/2144/2144.ts
function minimumCost(cost: number[]): number {
    let ans = 0;

    cost.sort((a, b) => b - a);

    for (let i = 0; i < cost.length; ++i) {
    if (i % 3 !== 2) {
      ans += cost[i];
    }
  }

  return ans;
};

File: 2144/2144.go
func minimumCost(cost []int) int {
	ans := 0

	sort.Sort(sort.Reverse(sort.IntSlice(cost)))

	for i := 0; i < len(cost); i++ {
		if i%3 != 2 {
			ans += cost[i]
		}
	}

	return ans
}